modules/planning/tasks/deciders/path_reference_decider/path_reference_decider.cc

```diff
@@ -162,7 +163,12 @@ Status PathReferenceDecider::Process(Frame *frame,
     return Status::OK();
   }
   std::vector<PathPoint> path_reference;
-  ConvertTrajectoryToPath(learning_model_trajectory, &path_reference);
+  // stiching
```

Comment:
misspelling

Reply:
maybe using  the expression like "adjusting trajectory init point" ? stitching is not an accurate word here

Reply:
Fixed. Thanks!